### ISSUE TITLE
Floating Objects removal

### ISSUE THREAD

**Issue by devu** [SEServerExtender/SEServerExtender#60]
After the Utilities tab was deleted there is no way to clear floating objects. To addition to that I am experiencing lots of issues related to them. 
- Objects seems to ignore maximum number set in config after a while.
- Number of them shows inaccurate number of objects in the world (Usually after restart you can see true picture and ads up even up to 1200 objects (on the world that max is set to 64)
- When objects overflow, being on the Entities tab is unresponsive and unable to manage grids. 
  Even entire app freezes. (Dual CPU Xenon E5 when CPU usage of the Extender is max 2%)

**Comment by EngineerBean** [comment]
there is a command that i find still works for me through SESE and thats '/delete all floatingobjects'
it does report that 0 have been deleted, but manages to clear them all anyway.

**Comment by devu** [comment]
Thanks for that tip.

Indeed it exposes all previously invisible/transparent for system objects,
freezes interface and then deletes them.
However is not changing the fact, it is no longer working as expected.

I only wonder what might be the issue behind it. Is it source of SE or SESE
or the fact something can't handle imported grids with something inside
inventory.
Things started to go weird when people were throwing some garbage away from
their cargo and mining at the same time.

**Comment by devu** [comment]
Well.. I was mistaken.

/delete all floatingobjects does not work either.

It only shows 0 on extender interface.

Logging into game world, using midspace admin commands to /collectobjects
radius will show a fountain of floating objects.

**Comment by dodexahedron** [comment]
If the extender GUI doesn't show the objects, then they are, in fact, gone from the server.
If your game client didn't hear about those updates, it wouldn't reflect those changes, and you'd have to disconnect and reconnect to notice it on your game client.

**Comment by devu** [comment]
Well, so you saying that using midspace admin command collectobejcts was in
fact happening on client only and that was a way to bring it back to server?

Because on the server I could not see them and after server restart (not a
client) I saw them all reveled.
Anyway after I managed to collect them all by hand using midspace tools
this is no longer happening.
It could be not related to SESE then. If so the ticket can be closed.
